Assets/Scripts/StaminaBar.cs

```diff
@@ -1,12 +1,23 @@
 ﻿using System.Collections;
 using System.Collections.Generic;
+using System.Linq;
 using UnityEngine;
 using UnityEngine.UI;
 
 public class StaminaBar : MonoBehaviour
 {
     public Image staminaFillImage;
+    public Image greyOutlineImage;
+    public GameObject glowOutline;
+    private Image glowOutlineImage;
+    public float glowSeconds = 4f;

```

Comment:
This isn't being used anymore right? We can remove this now right?

Reply:
Yes! took out glowoutlineimage